src/Select.soy

```diff
@@ -38,6 +40,7 @@
 			{param header kind="html"}
 				<button
 					class="{$buttonClass ? $buttonClass : ''} dropdown-select"
+					disabled="{$disabled}"
```

Comment:
Sure, the behavior we wanted for the this specific issue was just disabling the button. It'd be good to be able to disable the options too I guess, but less important, so we can open a separate issue for that.